src/main/java/org/fundacion/jala/converter/models/parameter/Parameter.java

```diff
@@ -7,13 +7,15 @@
  * license agreement you entered into with Fundacion Jala
  *
  * @author Juan Pablo Gonzales
- * @version 1.0
  */
 
 package org.fundacion.jala.converter.models.parameter;
 
 import org.springframework.stereotype.Service;
 
+/**
+ * This class is the options of the java parameters.
```

Comment:
The JavaDoc for this class is incorrect, is not related with Java, review that.

Reply:
ok 
it was solved